stock_release_channel/tests/test_release_channel.py

```diff
@@ -56,3 +56,15 @@ def test_default_sequence(self):
         self.assertEqual(channel2.sequence, 10)
         channel3 = self._create_channel(name="Test3")
         self.assertEqual(channel3.sequence, 20)
+
+    def test_is_manual_assignment(self):
```

Comment:
hi @jbaudoux here is the test of `is_manual_assignment`, please review it, thank you